Dockerfile/Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG ROS_DISTRO=humble
```

Comment:
Can we target rolling per default, please?

Reply:
I was thinking we can change this to rolling when pushing to the rolling branch. I had humble on my system so I developed for humble considering some humble code is running inside the container and I have to run rviz on my own system.

I could try to run the docker inside a docker container with rolling installed though.